packages/flutter/lib/src/painting/matrix_utils.dart

```diff
@@ -169,6 +169,66 @@ class MatrixUtils {
     transform = new Matrix4.copy(transform)..invert();
     return transformRect(transform, rect);
   }
+
+  /// Create a transformation matrix which mimics the effects of tangentially
+  /// wrapping the plane on which this transform is applied around a cylinder
+  /// and then looking at the cylinder from a point outside the cylinder.
+  ///
+  /// [radius] simulates the radius of the cylinder the plane is being wrapped
+  /// onto. If the transformation is applied to a 0-dimensional dot instead of
+  /// a plane, the dot would simply translate by +/- [radius] pixels along the
+  /// [orientation] [Axis] when rotating from 0 to +/- 90 degrees.
+  ///
+  /// A positive radius means the object is closest at 0 [angle] and a negative
+  /// radius means the object is closest at [pi] [angle].
```

Comment:
Does `pi` work here?
Check the output of the docs generator (e.g. look at the relevant Travis shard) to see if it causes an error.

Reply:
ah good catch. Was importing as math. I just typed π instead